frontend/src/pages/Admin.js

```diff
@@ -0,0 +1,59 @@
+import React, { Component } from 'react';
+import { connect } from 'react-redux';
+import { bindActionCreators } from 'redux';
+import { fetchAllCountStats } from '../actions/stats.js';
+import '../styles/App.scss';
+
+class AdminPage extends Component {
+  componentDidMount() {
+    this.props.fetchAllCountStats();
+  }
+  render() {
+    return (
+      <div>
+        <h1>Admin</h1>
+
+        {this.props.stats.count_stats_loading ? (
+          <p>Loading...</p>
+        ) : (
+          <div>
+            <div className="stats row">
+              <div className="card col-lg-3 col-md-5 m-2">
+                <div className="card-body">
```

Comment:
this is fine, but if you use reactstrap which is already installed, it can make the code a little cleaner since it basically wraps the messy bootstrap classnames e.g. https://reactstrap.github.io/components/card/